stdlib/mexpr/pprint.mc

```diff
@@ -150,16 +154,41 @@ lang FunPrettyPrint = FunAst
 end
 
 lang RecordPrettyPrint = RecordAst
+  sem record2tuple =
```

Comment:
Do we really want `record2tuple` as a semantic function? I tend to only use semantic functions for functions that might be extended in some other context (this might be controversial, though), and `record2tuple` seems very specific.

Concretly, my suggestion is to change this to:
```
let record2tuple = lam tm.
  use RecordAst in
  match tm with TmRecord t then
    ...
  else error "Not a record"
```

Reply:
That seems cleaner to me, I'll update!

Reply:
Now updated.